courts_db/data/courts.json

```diff
@@ -9828,7 +9890,9 @@
             "Illinois ${app}",
             "Illinois Court of Appeals",
             "Court of Appeals of Illinois",
-            "Appellate Court of Illinois Fifth District Industrial Commission ?(Division)?"
+            "Appellate Court of Illinois,? (First|Second|Third|Fourth|Fifth) District(\\.|,)? Industrial Commission Division",
+            "Appellate Court of Illinois,? (First|Second|Third|Fourth|Fifth) District(\\.|,)? Industrial Commission",
+            "Appellate Court of Illinois,? (First|Second|Third|Fourth|Fifth) District(\\.|,)? Workers' Compensation Commission Division"
```

Comment:
im not sure these are the same thing - they could be - but I think this requires more research